dataminer/Functions/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Style_guide.md

```diff
@@ -4,288 +4,210 @@ uid: LCA_Style_guide
 
-### Titles
+- If your app has more than 10 pages (including sidebar and header tabs), **add an overview page**. It should highlight key data and provide quick links.
 
-- Titles are optional on pages, depending on whether users will need more context information about what they are looking at.
+## Panels
 
-- Avoid repeating the app name as a title on an app's pages.
+- Show **additional information in a side panel on the right** for each item on a page.
 
-- Do not use a background color for titles.
+- If your panel updates based on a selection, **add a title** with the selected item's name.
 
-- Do not make the titles too big: the focus should be on the content of the page.
+- Show **forms in a pop-up** panel.
 
-- We recommend adding a title to a page as a web component instead of just text. For example:
+- Always choose the **same width** in your actions if you are using the same panel more than once.
 
-  - HTML code:
+- Every panel should have a **close button in the top-right corner**. An icon is enough; the button does not need a label.
 
-    ```html
-    <div style="height:100%;display:flex;flex-direction:column;justify-content:center">
-      <H1 style="color: rgb(20,33,42); font-family:'Segoe UI Light','Segoe UI Web Light','Segoe UI Web Regular','Segoe UI',Arial,sans-serif;margin:0;font-weight:100;font-size:24px">
-        <b>People</b>
-      </H1>
-      <p style="color: rgb(20,33,42,0.7); font-family:'Segoe UI Light','Segoe UI Web Light','Segoe UI Web Regular','Segoe UI','Segoe UI Symbol',HelveticaNeue-Light,'Helvetica Neue',Arial,sans-serif;margin:0;font-weight:600;font-size:16px">
-      An overview of all managed people
-      </p>
-    </div>
-    ```
+- If you have too much information in one panel, you can split it into multiple panels and **use the header bar as tabs**.
 
-  - Result:
+- **Use the “As overlay” option** when opening a panel.
 
-    ![Good title](~/dataminer/images/LCA_Style_Guide_Title_Good.png)
+- Do not use panels as pages.
 
-### Buttons
+## Actions
 
-[Buttons](xref:DashboardButton) can trigger different actions on a page or panel. Here are some things to keep in mind when you add buttons:
+- Only include **one access point** for a certain action per page/panel.
 
-- Use short labels.
+- **Confirm** important data change actions with a notification.
 
-- Add an icon to a button whenever possible. This will make it easier for users to see at a glance what the button is for.
+## Components
 
-- Make sure buttons have a transparent background.
+- Make sure you **choose the right visualization** for your data.
 
-   | 👍 Good | 👎 Bad |
-   |------|-----|
-   |![Good button](~/dataminer/images/LCA_Style_Guide_Button_Good.png)|![Bad button](~/dataminer/images/LCA_Style_Guide_Button_Bad.png)|
+- Make sure you add **[a custom empty component message](xref:Tutorial_Dashboards_Displaying_a_custom_empty_component_message)** that guides your user.
 
-### Images/icons
+### Title (Web component)
```

Comment:
What do you mean with "web component"? Instead of adding this as part of this header, it might be better to explain this below.